src/deploy.cmd.js

```diff
@@ -191,6 +191,12 @@ class DeployCommand extends AbstractCommand {
     this._target = path.resolve(this.directory, this._target);
     // init dev default file params
     this._default = Object.assign(this._defaultFile(this.directory), this._default);
+    // read the package json if present
+    try {
+      this._pkgJson = await fs.readJson(path.resolve(this.directory, 'package.json'));
+    } catch (e) {
+      this._pkgJson = {};
```

Comment:
Maybe we should add a test for this case: what happens if there is no `package.json` ?